Issue #769: [Fix] Group Task still has error status when Group Retry succeed
URL: https://github.com/treasure-data/digdag/pull/769

Commit ID: 8f39ef57fa58ebff2466dab9099b5de6b5daf7b6

Commit Message: Fix the query

Files Changed (1):
File: digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java ======================
    - Additions: 3
    - Deletions: 2
=======================================================

    ---Removed (old line 952):                       "select max(t.id) from tasks t join task_details td on t.id=td.id" +
    ---Removed (old line 953):                       " where parent_id = :parentId group by td.full_name" +
    +++Added (new line 952):                       "select max(t.id) from tasks t join task_details td on t.id = td.id" +
    +++Added (new line 953):                       " where parent_id = :parentId" +
    +++Added (new line 954):                       " group by td.full_name" +

